### ISSUE TITLE
VideoPress: not possible to set poster from video frame in Atomic sites

### ISSUE THREAD

**Issue by retrofox** [Automattic/jetpack#28460]
* Go to the block editor 
* Create a new block
* Upload a file
* Open the network tab
* Select an iframe by dragging the bullet of the range control

<img src="https://user-images.githubusercontent.com/77539/214232367-18d846b2-216c-4350-834b-395e492558d4.png" width="600px" />

* Search the poster request 

![Image](https://user-images.githubusercontent.com/77539/214230661-46541d3c-d93a-49a6-845e-30692154d216.png)

![Image](https://user-images.githubusercontent.com/77539/214230712-cea8e256-cb62-4737-a53c-c1c14d753b7c.png)

**Comment by lhkowalski** [comment]
I tried a lot of combinations of the plugin versions and was not able to replicate the issue. Things I did:

- created a new test site, upgraded it and made sure it was on Atomic
- enabled the beta extensions using the add_filter call on jetpack.php
- tested using the Jetpack version that shipped with the site (two minor versions behind, I guess) -> no issue
- added Jetpack Beta and update the Jetpack version to bleeding edge -> no issue
- installed Jetpack VideoPress 1.3 (last release) -> no issue
- updated Jetpack VideoPress to bleeding edge -> no issue

In all cases, the request to the poster endpoint returned a `200` status, both on the `POST` and the posterior `GET` polling requests.

@dhasilva also tried it and was not able to replica.

We will keep this open to make sure there was no catch on the testing that we did, but maybe the issue got solved by some other change from the past few days.
